### PR TITLE
differentiating node author comments 

### PR DESCRIPTION
**Pull request by katlazarova**

### CODE REVIEW

File: web/themes/custom/pineapple/components/_comments.twig [hunk 1]
```diff
@@ -83,10 +83,9 @@
     indicator here would break the render cache.
   #}
   <mark class="hidden new-label" data-comment-timestamp="{{ new_indicator_timestamp }}"></mark>
-
   {% set submitted_attributes = submitted_attributes|default(create_attribute()).addClass([
     'comment-submitted',
-    author_id == 1 ? 'admin-comment',
+    author_id == 1 or author_id == commented_entity.getOwnerId() ? 'highlighted-comment'
```
Comment (ElXyfer) [inline web/themes/custom/pineapple/components/_comments.twig:88]: Author id here, is this the user id of the author?

Comment (katlazarova) [reply web/themes/custom/pineapple/components/_comments.twig:88]: author ID is the user ID of the person writing the comment

Comment (ElXyfer) [reply web/themes/custom/pineapple/components/_comments.twig:88]: Okay so why are you using "1" as a part of this check?

Comment (katlazarova) [reply web/themes/custom/pineapple/components/_comments.twig:88]: Because that's the admin.

Comment (ElXyfer) [reply web/themes/custom/pineapple/components/_comments.twig:88]: Okay, why are you using the Id though, can you categorically guarantee you're only ever gonna have one admin on the site?

Comment (katlazarova) [reply web/themes/custom/pineapple/components/_comments.twig:88]: I was going to say yes, but actually, it doesn't matter. If the code doesn't accommodate the future possibilities it's not good enough, so I'll update this.

Comment (katlazarova) [reply web/themes/custom/pineapple/components/_comments.twig]: Done

File: web/themes/custom/pineapple/components/_comments.twig [hunk 2]
```diff
@@ -83,10 +83,10 @@
     indicator here would break the render cache.
   #}
   <mark class="hidden new-label" data-comment-timestamp="{{ new_indicator_timestamp }}"></mark>
-
+  {% set administrator = bamboo_has_role('administrator', author_id) ? 'TRUE' : 'FALSE' %}
```
Comment (ElXyfer) [inline web/themes/custom/pineapple/components/_comments.twig:86]: Looks like ```bamboo_has_role('administrator', author_id)``` already returns a bool value by default, so you don't need to set true or false yourself. You should remove that part and then on line 89 remove "== 'TRUE'"

Comment (katlazarova) [reply web/themes/custom/pineapple/components/_comments.twig]: Done.

**Pull request merged**